x-pack/platform/plugins/shared/onechat/server/services/agents/modes/utils/add_round_complete_event.ts

```diff
@@ -67,14 +72,30 @@ export const addRoundCompleteEvent = ({
 const createRoundFromEvents = ({
   events,
   input,
+  startTime,
+  endTime,
```

Comment:
NIT: `endTime = new Date()` here instead of 

```
  if (!endTime) {
    endTime = new Date();
  }
```

below ?

Reply:
https://github.com/elastic/kibana/pull/241681/commits/1536a2a5dd1642d8fdc0139b250dfc79fa7e6f3c